terminal/command.go

```diff
@@ -174,7 +176,7 @@ func threads(client service.Client, args ...string) error {
 
 func thread(client service.Client, args ...string) error {
 	if len(args) == 0 {
-		return fmt.Errorf("you must specify a thread")
+		return printscope(client)
```

Comment:
Is the reasoning for this change that `thread` should just return current thread info if one is not specified? Because this function allows you to switch threads.